### ISSUE TITLE
Waiting for selector `div[aria-live=polite] span` failed

### ISSUE THREAD

**Issue by huseinzol05** [Songkeys/Translateer#13]
```
{"statusCode":500,"error":"Internal Server Error","message":"Waiting for selector `div[aria-live=polite] span` failed: Waiting failed: 30000ms exceeded"}
```

I tried on 3 different server location, got the same error, I guess Google Translate patched the html?

**Comment by huseinzol05** [comment]
convert to,

```
const targetSelector = `span[lang=${to}]`;
```

solved the problem, but not sure for how long.

**Comment by songkeys** [comment]
I noticed that it changed the UI a lot. I'll try fix this later when I'm free.
